beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl

```diff
@@ -707,12 +713,25 @@ workflow GDCWholeGenomeSomaticSingleSample {
         input: input_bam = picard_markduplicates.bam
     }
 
+    call CheckContamination.CalculateSomaticContamination as check_contamination {
+        input:
+            reference = ref_fasta,
+            reference_dict = ref_dict,
+            reference_index = ref_fai,
+            tumor_cram_or_bam = sort_and_index_markdup_bam.bam,
+            tumor_crai_or_bai = sort_and_index_markdup_bam.bai,
+            contamination_vcf = contamination_vcf,
+            contamination_vcf_index = contamination_vcf_index
+    }
+
     call gatk_baserecalibrator {
         input:
             bam = sort_and_index_markdup_bam.bam,
             ref_fasta = ref_fasta,
             ref_fai = ref_fai,
-            ref_dict = ref_dict
+            ref_dict = ref_dict,
+            dbsnp_vcf = dbsnp_vcf,
```

Comment:
Is this related to the contamination change?  Or just an improvement?

Reply:
It is not related. These are required inputs to the task and were being passed from the inputs file directly to the task. I preferred to pass them to the pipeline inputs and then to the task, since I don't like the concept of having required inputs with no defaults left out of the top level pipeline inputs